### PR TITLE
Ap/show unresolved

### PR DESCRIPTION
**Pull request by abpiskunov**
**Customer scenario**

Changing package version from known to unknown version causes unresolved dependency to disappear from solution explorer

**Bugs this fixes:**

https://github.com/dotnet/sdk/issues/451

**Workarounds, if any**

Reload project

**Risk**

Minimal

**Performance impact**

none

### CODE REVIEW

**Comment by abpiskunov** [comment]
@dotnet/project-system @srivatsn @natidea

File: src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/Tree/Dependencies/NuGetDependenciesSubTreeProviderTests.cs [hunk 1]
```diff
@@ -184,7 +184,7 @@ public class NuGetDependenciesSubTreeProviderTests
                 ""ItemSpec"": ""tfm2/ExistingUnresolvedPackage/2.0.0"",
                 ""ItemType"": ""PackageReference""
             }
-        }
+        }
```
Comment (srivatsn) [inline src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/Tree/Dependencies/NuGetDependenciesSubTreeProviderTests.cs]: whitespace

**Comment by srivatsn** [comment]
@MattGertz for RTM consideration - If someone edits the project file to change the version of a package, the node disappears from the solution tree instead of showing up as unresolved with a warning sign.

**Comment by MattGertz** [comment]
As bar check.

**Comment by srivatsn** [comment]
This was rejected in shiproom. I'm retargeting this PR to the post-dev15 branch.

**Comment by srivatsn** [comment]
You'll need to rebase.

**Comment by basoundr** [comment]
@abpiskunov Could you please fix the conflicts and update the PR?

**Comment by abpiskunov** [comment]
this PR is obsolete. There is a PR https://github.com/dotnet/roslyn-project-system/pull/1880/files that addresses this issue in the new  design

**Comment by 333fred** [comment]
Closing then.

**Pull request closed without merging**